### ISSUE TITLE
Chrome crashed in undetected-chromedriver mode

### ISSUE THREAD

**Issue by M-Zubair10** [seleniumbase/SeleniumBase#1586]
Getting this error in seleniumbase with uc while it works fine with chromedriver simple or undetected-chromedriver 

(unknown error: DevToolsActivePort file doesn't exist)
  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)
Stacktrace:
#0 0x55a9f2ec9693 <unknown>
#1 0x55a9f2cc2b0a <unknown>
#2 0x55a9f2ce7ef2 <unknown>
#3 0x55a9f2ce31f4 <unknown>
#4 0x55a9f2d1e953 <unknown>
#5 0x55a9f2d18743 <unknown>
#6 0x55a9f2cee533 <unknown>
#7 0x55a9f2cef715 <unknown>
#8 0x55a9f2f197bd <unknown>
#9 0x55a9f2f1cbf9 <unknown>
#10 0x55a9f2efef2e <unknown>
#11 0x55a9f2f1d9b3 <unknown>
#12 0x55a9f2ef2e4f <unknown>
#13 0x55a9f2f3cea8 <unknown>
#14 0x55a9f2f3d052 <unknown>
#15 0x55a9f2f5771f <unknown>
#16 0x7f737ab4f6db <unknown>

**Comment by mdmintz** [comment]
Could be https://github.com/ultrafunkamsterdam/undetected-chromedriver/issues/353
Could be https://github.com/ultrafunkamsterdam/undetected-chromedriver/issues/137
Could be https://github.com/ultrafunkamsterdam/undetected-chromedriver/issues/118
Could be https://github.com/ultrafunkamsterdam/undetected-chromedriver/issues/123

I need more info to debug. What OS & version? Are you trying to open up more browsers than available CPUs? Are you changing any other configuration? It works for me with default configuration.

**Comment by M-Zubair10** [comment]
removing userdatadir works for me

**Comment by mdmintz** [comment]
I'll have updates in ``seleniumbase`` ``4.7.1`` that should solve that issue. I'm working on completing that.

**Comment by M-Zubair10** [comment]
Do you know any chrome flag to open chrome at secondary ip or nic

I attached 10 ms public ips to vm, I want to open 10 unique chromes each
with one of the ip but it always read my default first ip

**Comment by M-Zubair10** [comment]
In linux

**Comment by mdmintz** [comment]
@M-Zubair10 

For changing the ``ip`` of the browser, run tests using:

```bash
pytest --proxy=server:port
```

Or authenticated:

```bash
pytest --proxy=user:pass@server:port
```

You can use test parameterization to turn one test into many: https://github.com/seleniumbase/SeleniumBase/blob/master/examples/parameterized_test.py

And you can spin up new drivers with proxy settings during tests, eg:

```python
self.get_new_driver(uc=True, proxy="server:port")
```

If using ``--wire`` mode, you can change your proxy in the middle of the test:

```python
self.set_wire_proxy("server:port")  # Requires "--wire"!
```

But note that you cannot use both ``--uc`` and ``--wire`` at the same time. Pick one.

**Comment by M-Zubair10** [comment]
No, no
I know this, I am talking about my ip addresses attached to virtual machine
Primary ip address, secondary ip address... 10
Chrome always use primary ip address

Thanks a lot

**Comment by mdmintz** [comment]
If there are multiple addresses attached to a virtual machine, then maybe you can use the proxy settings to switch to each one individually. I'm not sure about browsers magically picking up the proxy for each of those unless you set it to use it.

**Comment by M-Zubair10** [comment]
How to use ip adress as proxy, i mean how to convert it in linux

**Comment by mdmintz** [comment]
I would try searching Stack Overflow. If you have a specific ip address already for setting a browser's proxy to, then it's easy with SeleniumBase at that point to change the proxy settings.

**Comment by M-Zubair10** [comment]
Its nic ip addresses, these are static and I can attach upto 250 ip to
single network interface, I just don't know how to configure it

One of my ip is 20.89.0.14

**Comment by mdmintz** [comment]
Released in ``4.7.1`` - https://github.com/seleniumbase/SeleniumBase/releases/tag/v4.7.1
